python/CCLE_data.py

```diff
@@ -20,16 +20,20 @@
 from scipy.stats.stats import pearsonr
 import matplotlib.pyplot as plt
 import seaborn as sns
-
-
+import plotly.io as pio
+#import plotly.graph_objects as go
+import plotly.figure_factory as ff
+from scipy.spatial.distance import pdist, squareform
 
 
 
 """
 FUNCTIONS
 """
-#Used to find common celllines
-def common_celllines(a,b):
+
+def common(a,b):
+    """ This functions purpose is to find common elements of lists by converting
```

Comment:
Example of how you should write all your docstrings:

"""
common finds all common elements between two lists

INPUTS:
  * a: a list
  * b: another list

OUTPUTS:
  * common: a set containing common elements between list a and list b
"""